---
layout: default_modified
---
Research assistant professor working with [Dassanayake lab.@LSU](https://www.lsugenomics.org/). I study plant comparative genomics, especially for those found in "extreme" environments a.k.a. [extremophytes](https://extremeplants.org/what-is-an-extremophyte/).

___
## News 

- 2021-11-28: The first draft genome of _Phragmites australis_ (common reed) was published in [_Mol Ecol_](https://doi.org/10.1111/mec.16293) :) Fun collaboration with researchers from Tulane Univ. and USGS. Next: chromosome-level assemblies of both native and invasive (in North America) subspecies and more comparative genomics of _Phragmites_ invasion.

___
## Research

- Comparative genomics of [_Schrenkiella parvula_](https://extremeplants.org/species/schrenkiella-parvula/?ms=halophytes) and [_Eutrema salsugineum_](https://extremeplants.org/species/eutrema-salsugineum/?ms=halophytes), extreme stress-adapted relatives of _Arabidopsis_ and Brassicaceae crops (e.g. Camelina, canola, and other Brassica crops)

- Comparative genomics of [invasive and native _Phragmites australis_ (common reed) in North America](https://nas.er.usgs.gov/queries/greatlakes/FactSheet.aspx?Species_ID=2937)

- Comparative genomics of seagrass [_Halophila stipulacea_](https://www.gidon-winters.com/research) and its freshwater relative [_Vallisneria americana_](https://plants.ifas.ufl.edu/plant-directory/vallisneria-americana/)

- Developing tools to systemetically detect lineage(s)-specific modifications in gene copy numbers and gene regulatory networks (GRNs)

Details under construction ...

___
## Random thoughts, interests, and fantasies
<details>
<summary> reveal ... </summary>

- How to identify significant (e.g. adaptive) GRN modifications in recent polyploids (e.g. _Camelina sativa_) compared to their diploid relatives. Or is it even doable/useful?

- How to compare GRNs among multiple species effectively without relying on "reciprocal 1-vs-1 best-hit" ortholog pairs, including all lineage(s)-specific duplicates?

- A reference genome can be a chimera of haplotypes and whatnot, if the source is not only heterogygous but also heterogeneous (e.g. ecological samples). Would it be more useful as a reference to allow "wobble" nucleotides (then how about SVs ...)?

- How would the (epi-)genome of a Tolkeinian elf (e.g. Legolas) be different from human? [#ElvenGenomics](https://twitter.com/inspirace/status/1467178107018915846?s=20)?

</details>

___
## [Tools](tools.md)

___
## [Publications](publications.md)

___
## [Grants](grants.md)
