---
layout: default2
---
Research assistant professor working with [Dassanayake lab.@LSU](https://www.lsugenomics.org/). I study plant comparative genomics, especially for those found in "extreme" environments a.k.a. [extremophytes](https://extremeplants.org/what-is-an-extremophyte/).

## Research
- Comparative genomics of [_Schrenkiella parvula_](https://extremeplants.org/species/schrenkiella-parvula/?ms=halophytes) and [_Eutrema salsugineum_](https://extremeplants.org/species/eutrema-salsugineum/?ms=halophytes), extreme stress-adapted relatives of _Arabidopsis_ and Brassicaceae crops (e.g. Camelina, canola, and other Brassica crops)
- Comparative genomics of [invasive and native _Phragmites australis_ (common reed) in North America](https://nas.er.usgs.gov/queries/greatlakes/FactSheet.aspx?Species_ID=2937)
- Comparative genomics of seagrass [_Halophila stipulacea_](https://www.gidon-winters.com/research) and its freshwater relative [_Vallisneria americana_](https://plants.ifas.ufl.edu/plant-directory/vallisneria-americana/)
- Developing tools to systemetically detect lineage(s)-specific modifications in gene copy numbers and gene regulatory networks (GRNs)

Details under construction ...

## Tools
- [OrthNet](https://github.com/ohdongha/OrthNet): an attempt to identify co-linearity and ortholog groups, and encode evolutionary histories as the topology of Ortholog Networks (OrthNet), among multiple closely related genomes.
- [Genome toolbox](https://github.com/ohdongha/Genome-Toolbox): small python scripts useful for manipulating genome assembly (.fa) and annotation (.gtf) files
- PiP (under construction): an attempt to identify functions and gene sets (e.g. Gene Ontology terms) that underwent lineage(s)-specific modifications in gene expression/regulation among multiple species, by performing Phylogenetically-informed Profiling (PiP) of gene sets.

## [Publications](publications.md)

## Grants and awards
Under construction ...
