---
layout: default_modified
---

# Random questions and fantasies

___
### 2022 Jan.

- What are the better measures of conservation or modification in gene regulation between sets of ortholog pairs, that account for complex expression datasets such as time-series or tissue/cell type-specific expression.

- How to identify significant (e.g. adaptive) GRN modifications from background divergence derived from genetic drift.

- How to identify significant (e.g. adaptive) GRN modifications in recent polyploids (e.g. _Camelina sativa_) compared to their diploid relatives. Or is it even doable/useful?

- How to compare GRNs among multiple species effectively without relying on "reciprocal 1-vs-1 best-hit" ortholog pairs, including all lineage(s)-specific duplicates?

- A reference genome can be a chimera of haplotypes and whatnot, if the source is not only heterogygous but also heterogeneous (e.g. ecological samples). Would it be more useful as a reference to allow "wobble" nucleotides (then how about SVs ...)?

- How would the (epi-)genome of a Tolkeinian elf (e.g. Legolas) be different from human? [#ElvenGenomics](https://twitter.com/inspirace/status/1467178107018915846?s=20)?

___
Always click the large [Dong-Ha Oh](index.md) to go back to main index