---
layout: default_modified
---

# Random questions

## Potentially relevant readings
___

### 2022 Aug.

- What are the repeat-masking methods that are computationally less expensive than RepeatModler and RepeatMasker, when your goal is masking repetitive sequences to facilitate the whole genome alignment (i.e. you are not interested in the classification, etc. of repeat elements, and a bit of "over-masking" is okay)? 

	- Hausmann and Kurtz (2021) DeepGRP: engineering a software tool for predicting genomic repetitive elements using Recurrent Neural Networks with attention [_Algorithms Mol Biol_ 16:20](https://doi.org/10.1186/s13015-021-00199-0){:target="_blank"}
	- Feng, _et al_. (2021) Sequence repetitiveness quantification and _de novo_ repeat detection by weighted k-mer coverage [_Brief Bioinform_ 22:bbaa086](https://doi.org/10.1093/bib/bbaa086){:target="_blank"}
	- Osipova, Hecker, and Hiller (2019) RepeatFiller newly identifies megabases of aligning repetitive sequences and improves annotations of conserved non-exonic elements  [_Gigascience_ 8:giz132](https://doi.org/10.1093/gigascience/giz132){:target="_blank"}
	- Mori, _et al_. (2019) Fast and global detection of periodic sequence repeats in large genomic resources [_Nucleic Acids Res_ 47:e8](https://doi.org/10.1093/nar/gky890){:target="_blank"}
	- Lian, _et al_, (2016) A Complete and Accurate Ab Initio Repeat Finding Algorithm [_Interdiscip Sci_ 8:75-83](https://doi.org/10.1007/s12539-015-0119-6){:target="_blank"}
	- Girgis (2016) Red: an intelligent, rapid, accurate tool for detecting repeats de-novo on the genomic scale [_BMC Bioinformatics_ 16:227](https://doi.org/10.1186/s12859-015-0654-5){:target="_blank"} 

(Back to [questions](random.md) or [bookmarks](bookmarks.md))

___
Always click the large [Dong-Ha Oh](index.md) to go back to main index
