---
layout: default_modified
---

# Random questions

## Potentially relevant readings
___

### 2022 Jan.

- What are the better measures of conservation or modification in gene regulation between sets of ortholog pairs, that account for complex expression datasets such as time-series or tissue/cell type-specific expression?

	- Patel, et al. (2012) BAR Expressolog Identification: Expression Profile Similarity Ranking of Homologous Genes in Plant Species. [_Plant J_ 71:1038–50](https://doi.org/10.1111/j.1365-313X.2012.05055.x){:target="_blank"}
	- Das, et al. (2016) Expression Pattern Similarities Support the Prediction of Orthologs Retaining Common Functions after Gene Duplication Events. [_Plant Physiol_ 171:2343–57](https://doi.org/10.1104/pp.15.01207){:target="_blank"}
	- Lieberman-Lazarovich, et al., (2019) Deep Conservation of Cis-Element Variants Regulating Plant Hormonal Responses. [_Plant Cell_ 31:tpc.00129.2019](https://doi.org/10.1105/tpc.19.00129){:target="_blank"}
	- Lambert, et al. (2019) Similarity Regression Predicts Evolution of Transcription Factor Sequence Specificity. [_Nat Genet_ 51:981–89](https://doi.org/10.1038/s41588-019-0411-1){:target="_blank"}
	- Lee, et al. (2019) Comparing Time Series Transcriptome Data between Plants Using a Network Module Finding Algorithm. [_Plant Methods_ 15:61](https://doi.org/10.1186/s13007-019-0440-x){:target="_blank"}
	- Jones, et al. (2020) Identification and Evolution of Gene Regulatory Networks: Insights from Comparative Studies in Plants. [_Curr Opi Plant Biol_ 54:42–48](https://doi.org/10.1016/j.pbi.2019.12.008){:target="_blank"}
	- Lee, et al. (2020) CoCoCoNet: Conserved and Comparative Co-Expression across a Diverse Set of Species. [_Nucleic Acids Res_ 48:W566–71](https://doi.org/10.1093/nar/gkaa348){:target="_blank"}
	- Ezoe, et al. (2021) Degree of Functional Divergence in Duplicates Is Associated with Distinct Roles in Plant Evolution. [_Mol Biol Evol_ 38:1447–59](https://doi.org/10.1093/molbev/msaa302){:target="_blank"}
	- Shew, et al. (2021) Diverse Molecular Mechanisms Contribute to Differential Expression of Human Duplicated Genes. [_Mol Biol Evol_ 38:3060–77](https://doi.org/10.1093/molbev/msab131){:target="_blank"}
	- Reshef, et al. (2021) Co-Varying Neighborhood Analysis Identifies Cell Populations Associated with Phenotypes of Interest from Single-Cell Transcriptomics. [_Nat Biotechnol_](https://doi.org/10.1038/s41587-021-01066-4){:target="_blank"}
	- Ovens, et al. (2021) Comparative Analyses of Gene Co-Expression Networks: Implementations and Applications in the Study of Evolution. [_Front Genet_ 12:695399](https://doi.org/10.3389/fgene.2021.695399){:target="_blank"}
	- Ovens, et al. (2021) Juxtapose: A Gene-Embedding Approach for Comparing Co-Expression Networks. [_BMC Bioinformatics_ 22:125](https://doi.org/10.1186/s12859-021-04055-1){:target="_blank"}

(Back to [questions](random.md) or [bookmarks](bookmarks.md))

___
Always click the large [Dong-Ha Oh](index.md) to go back to main index